client/state/action-types.js

```diff
@@ -14,33 +14,35 @@ export const ANALYTICS_EVENT_RECORD = 'ANALYTICS_EVENT_RECORD';
 export const ANALYTICS_MULTI_TRACK = 'ANALYTICS_MULTI_TRACK';
```

Comment:
Did you order those constants manually or do you use anything fancy to do it for you? :)